.gitignore

```diff
@@ -31,6 +31,7 @@ DerivedData
 *.ipa
 *.dSYM.zip
 *.dSYM
+*.dia
```

Comment:
These files get spewed out when compiling with the development toolchain so I added them to the ignore file.